src/tabmat/standardized_mat.py

```diff
@@ -130,7 +130,15 @@ def sandwich(
         if not hasattr(d, "dtype"):
             d = np.asarray(d)
         check_sandwich_compatible(self, d)
-
+        # stat_mat = mat * mult[newaxis, :] + shift[newaxis, :]
+        # stat_mat.T @ d[:, newaxis] * stat_mat
+        # = mult[:, newaxis] * mat.T @ d[:, newaxis] * mat * mult[newaxis, :] +   (1)
+        #   mult[:, newaxis] * mat.T @ d[:, newaxis] * np.outer(ones, shift) +    (2)
+        #   shift[:, newaxis] @ d[:, newaxis] * mat * mult[newaxis, :] +          (3)
+        #   shift[:, newaxis] @ d[:, newaxis] * shift[newaxis, :]                 (4)
+        #
+        # (1) = self.mat.sandwich(d) * np.outer(limited_mult, limited_mult)
+        # (2) = mult * self.transpose_matvec(d) * shift[newaxis, :]
```

Comment:
Is this comment meant to be added to the code? In that case, I could use some context here 😅

Reply:
This is similar to the comment in `StandardizedMatrix.transpose_matvec`. I needed to write this down. We can also remove this.